src/include/kdump/dialogs.rb

```diff
@@ -541,6 +522,58 @@ def initialize_kdump_dialogs(include_target)
       }
     end
 
+    def kdump_memory_widget
+      min = Kdump.memory_limits[:min_low].to_i
+      max = Kdump.memory_limits[:max_low].to_i
+      if Kdump.high_memory_supported?
+        low_label = _("Kdump &Low Memory [MB]")
+      else
+        low_label = _("Kdump Memor&y [MB]")
+      end
+      low_label += " (#{min} - #{max})"
+      widgets = [
+        Left(
+          HBox(
+            Left(Label(_("Total System Memory [MB]:"))),
```

Comment:
Whatever kdumptool reports. I think it's MiB in this case. And since we have to adapt the translations anyway, it makes sense to fix it.